tensorflow/cc/framework/cc_op_gen.cc

```diff
@@ -57,6 +57,16 @@ string GetPath(const string& dot_h_fname) {
   return result;
 }
 
+// Converts: some/path/to/file.xx
+// to: file
+// (note that suffix is removed)
+string GetFilename(const string& path) {
```

Comment:
These are code changes to the part of the code which writes the comments for autogenerated files, which in turn show up in the docs. @josh11b, @skye for context.